lib/widgets/action_sheet.dart

```diff
@@ -647,7 +647,7 @@ class UnsubscribeButton extends ActionSheetMenuItemButton {
       final zulipLocalizations = ZulipLocalizations.of(pageContext);
 
       final dialog = showSuggestedActionDialog(context: pageContext,
-        title: zulipLocalizations.unsubscribeConfirmationDialogTitle(subscription.name),
+        title: zulipLocalizations.unsubscribeConfirmationDialogTitle('#${subscription.name}'),
```

Comment:
This should get a matching change in the example for this placeholder in `app_en.arb`.